test/unit/vm/json_schema/invalid_schemas_test.dart

```diff
@@ -41,31 +41,34 @@ library json_schema.test_invalid_schemas;
 import 'dart:convert' as convert;
 import 'dart:io';
 import 'package:json_schema/json_schema.dart';
+import 'package:json_schema/vm.dart';
```

Comment:
Any desire to separate the imports here and other places in the repo or just keep them together?